### ISSUE TITLE
Bulk Uploading

### ISSUE THREAD

**Issue by guandai** [like-minded-dk/group-web-shop#33]

**Comment by guandai** [comment]
there is a bug  I am fixing

**Comment by guandai** [comment]
@ningtandk  @christopherfrandsen   please test this link from your side.  if it works we can move it to done

**Comment by ningtandk** [comment]
yes, I can see them here on the backend, but I guess they should also be shown on the wp.like-minde.dk? @guandai

**Comment by guandai** [comment]
@guandai guandai here is a message

**Comment by guandai** [comment]
I create a page to test  bulk uploading 
https://wp.like-minded.dk/index.php/2024/01/26/test-gallery-page/



![Image](https://github.com/like-minded-dk/group-web-shop/assets/1208517/fde8b5fb-3674-4d00-a394-d6f813061108)

![Image](https://github.com/like-minded-dk/group-web-shop/assets/1208517/6d3f176f-ac49-4682-8b67-50aae06e555b)
